app/controllers/support_interface/application_forms/email_subscription_controller.rb

```diff
@@ -0,0 +1,36 @@
+module SupportInterface
+  module ApplicationForms
+    class EmailSubscriptionController < SupportInterfaceController
+      before_action :set_application_form
+
+      def edit
+        @email_subscription_form = EmailSubscriptionForm.build_from_application(@application_form)
+      end
+
+      def update
+        @email_subscription_form = EmailSubscriptionForm.new(unsubscribed_from_emails_params)
+
+        if @email_subscription_form.save(@application_form)
+          flash_success
+          redirect_to support_interface_application_form_path(@application_form)
+        else
+          render :edit
```

Comment:
```suggestion
          render :edit, status: :unprocessable_entity
```
Not sure we do this everywhere but we should 👌

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/10323/commits/c60ba88dad3c9dcd8c525dae44dcf4f924707ca4